rosbridge_server/scripts/rosbridge_websocket_client.py

```diff
@@ -34,10 +36,12 @@ class WebsocketClientTornado():
     keepalive = None
 
     def __init__(self, uri):
+        self.authenticated = False
```

Comment:
Yes, I know, this is a temporary solution until sessions for each client are used.